apps/src/studio/studio.js

```diff
@@ -1709,6 +1709,14 @@ Studio.reset = function(first) {
   Studio.sayComplete = 0;
   Studio.playSoundCount = 0;
 
+  // More things used to validate level completion.
+  Studio.trackedBehavior = {
```

Comment:
where does `trackedBehavior` get reset back to these values? That seems important.